src/components/Filmchoice.jsx

```diff
@@ -2,34 +2,56 @@ import React from 'react';
 import './Filmchoice.css';
 import axios from 'axios';
 import apiKey from './apiKey';
+import FilmZoom from './FilmZoom';
 
 class Filmchoice extends React.Component {
   constructor(props) {
     super(props);
-    this.state = { value: '', results: [] };
+    this.state = {
+      value: '',
+      results: [],
+      like: [],
+      clicked: 'view',
+      choosenOne: 'invisible',
+    };
     this.movieSearch = this.movieSearch.bind(this);
   }
-  /** deuxième clès resolve ou movie
+  /** deuxième clès resolve ou elect
    * intialiser avec un tableau vide
    * enturer formulaire ave div  et map sur le tableau dans le state
    */
 
   fetchMovie = (event) => {
     event.preventDefault();
+    this.setState({ clicked: 'view' });
     const { value } = this.state;
     const url = `https://api.themoviedb.org/3/search/movie?query=${value}&api_key=${apiKey}`;
     axios.get(url).then((response) => {
       this.setState({ results: response.data.results });
     });
```

Comment:
Could you add somewhere the filter on movies that have no poster ?  .filter((moviecover) => moviecover.poster_path != null)